mixin-backports.php

```diff
@@ -35,6 +35,14 @@
     'provided-by' => '5.45.beta1',
     'minimum' => '5.27', /* Compat may go back further; haven't tested */
   ],
+  'entity-types-php@1' => [
+    'version' => '1.0.0',
+    'sha256' => 'f8e10aac991b2b3acac269a1fca81f883a295908990687959e790885a2e410c2',
+    'remote' => 'https://raw.githubusercontent.com/civicrm/civicrm-core/8de7881b35e3f98d38b147dcf23f5a44d5dfe035/mixin/entity-types-php%401/mixin.php',
```

Comment:
@totten so this url will be prettier once https://github.com/civicrm/civicrm-core/pull/24947 is merged right? IMO let's go ahead and merge it.

Reply:
Ok merged. This URL can be updated.